requirements/app/base.txt

```diff
@@ -8,7 +8,7 @@ py
 starsessions
 requests
 urllib3
-fsspec==2022.01.0
+fsspec==2022.5.0
```

Comment:
Just pointing out that the version is fixed with `==`. (It's fine if it's expected by the app side.)